skxray/core/roi.py

```diff
@@ -430,11 +432,23 @@ def circular_average(image, calibrated_center, threshold=0, nx=100,
     ring_averages : array
         Radial average of the image. shape is (nx, ).
     """
-    radial_val = utils.radial_grid(calibrated_center, image.shape,
-                                   pixel_size)
+    radial_val = utils.radial_grid(calibrated_center, image.shape, pixel_size)
+
+    if min_x is None:
```

Comment:
I don't think you need this logic, it is duplicated in https://github.com/tacaswell/scikit-xray/blob/master/skxray/core/utils.py#L588